reference/api/search.md

```diff
@@ -70,11 +72,23 @@ Query terms enclosed in double quotes are treated as [phrase searches](#query-q)
 | **`hits`**               | Array of objects | Results of the query                            |
 | **`offset`**             | Number           | Number of documents skipped                     |
 | **`limit`**              | Number           | Number of documents to take                     |
-| **`estimatedTotalHits`** | Number           | Total number of matches                         |
+| **`estimatedTotalHits`** | Number           | Estimated total number of matches               |
+| **`totalHits`**          | Number           | Exhaustive total number of matches              |
+| **`totalPages`**         | Number           | Exhaustive total number of search results pages |
```

Comment:
Not sure about this one. Should it be "search **result** pages" or "search **results** pages"
```suggestion
| **`totalPages`**         | Number           | Exhaustive total number of search result pages |
```